x-pack/plugins/security_solution/public/management/components/artifact_entry_card/hooks/use_endpoint_policies_to_artifact_policies.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+import { MenuItemPropsByPolicyId } from '..';
+import { PolicyData } from '../../../../../common/endpoint/types';
+import { useAppUrl } from '../../../../common/lib/kibana';
+import { getPolicyDetailPath } from '../../../common/routing';
+
+/**
+ * Takes a list of EndpointPolicies (PolicyData) and turn them
+ * into MenuItemPropsByPolicyId required by the artifact card.
+ *
+ * The resulting menu will open the policies in a new tab
+ *
+ */
+export const useEndpointPoliciesToArtifactPolicies = (
```

Comment:
@paul-tavares I turned this into a reusable hook since the functionality is simply open it on a new tab.